src/core/webidl-index.js

```diff
@@ -42,9 +42,11 @@ export function run(conf, doc, cb) {
   Array.from(document.querySelectorAll("pre.def.idl"))
     .map(elem => {
       const span = document.createElement("span");
-      const clone = elem.cloneNode(true).firstElementChild;
-      span.appendChild(clone);
-      span.appendChild(document.createTextNode("\n"));
+      const children = elem.cloneNode(true).children;
+      Array.from(children).forEach(child => {
```

Comment:
Can we use the new for-of syntax? `for (const child of Array.from(children))`

Reply:
We’ve been using mostly functional style, but here it might make sense as it’s a single `forEach`.